sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -178,7 +178,12 @@ internal class RadarApiClient(
         val anonymous = RadarSettings.getAnonymousTrackingEnabled(context)
         try {
             params.putOpt("anonymous", anonymous)
-            if (!anonymous) {
+            if (anonymous) {
+                params.putOpt("geofenceIds", RadarState.getGeofenceIds(context)?.toTypedArray())
```

Comment:
There's no `putArray()` on `SharedPreferences`, so we need to store these as `Set<String>`, and call `toTypedArray()` when sending to `POST /track`